package.json

```diff
@@ -35,34 +37,35 @@
       "value": "http://localhost:1337/stub-content/index.html"
     },
     {
-        "name": "BADGE_COLOR",
-        "title": "Ideatown Badge Color",
-        "description": "Color of the ideatown badge hex",
-        "type": "string",
-        "value": "#00AAAA"
+      "name": "BADGE_COLOR",
+      "title": "Ideatown Badge Color",
+      "description": "Color of the ideatown badge hex",
+      "type": "string",
+      "value": "#00AAAA"
+    },
+    {
+      "name": "ALLOWED_ORIGINS",
+      "title": "Allowed origins for addon",
+      "description": "Allowed origins for addon",
+      "type": "string",
+      "value": "http://localhost:1337/*,http://127.0.0.1:1337/*,https://ideatown.firefox.com/*,https://mozilla.github.io/idea-town-addon/stub-content/,https://people.mozilla.org/~jhirsch/universal-search-addon/*"
+    }
+  ],
+  "UPDATES": [
+    {
+      "title": "Universal Search",
+      "desc": "Expandable embed search from your address bar",
+      "url": "https://people.mozilla.org/~jhirsch/universal-search-addon/addon.xpi"
+    },
+    {
+      "title": "Fox Splitter",
+      "desc": "Split your browser windows like a tiling wm",
+      "url": "https://addons.mozilla.org/firefox/downloads/latest/324976/addon-324976-latest.xpi?src=search"
     },
     {
-        "name": "ALLOWED_ORIGINS",
-        "title": "Allowed origins for addon",
-        "description": "Allowed origins for addon",
-        "type": "string",
-        "value": "http://localhost:1337/*,http://127.0.0.1:1337/*,https://ideatown.firefox.com/*,https://mozilla.github.io/idea-town-addon/stub-content/,https://people.mozilla.org/~jhirsch/universal-search-addon/*"
-    }],
-    "UPDATES": [
-      {
-        "title": "Universal Search",
-        "desc": "Expandable embed search from your address bar",
-        "url": "https://people.mozilla.org/~jhirsch/universal-search-addon/addon.xpi"
-      },
-      {
-        "title": "Fox Splitter",
-        "desc": "Split your browser windows like a tiling wm",
-        "url": "https://addons.mozilla.org/firefox/downloads/latest/324976/addon-324976-latest.xpi?src=search"
-      },
-      {
-        "title": "Idea Town",
-        "desc": "Manage idea town experiments",
-        "url": "http://localhost:1337/dist/idea-town-addon-0.0.1.xpi"
-      }
-    ]
+      "title": "Idea Town",
+      "desc": "Manage idea town experiments",
+      "url": "http://localhost:1337/dist/idea-town-addon-0.0.1.xpi"
+    }
+  ]
```

Comment:
Sorry for the noise. It seems doing an `npm i eslint-config-idea-town` causes a full auto-format of the package.json file, so this indenting got a bit tweaked. As always, you can peep the diff without whitespace changes using https://github.com/mozilla/idea-town-addon/pull/14/files?w=0